app/scripts/directives/header/style.css

```diff
@@ -22,3 +22,8 @@ dashboard-header .option {
   padding: 10px;
 }
 
+md-icon .icon {
```

Comment:
You need to remove the space between the two classes for the style to be applied, It is currently not being applied as the attribute/class being targeted are on the same element.